sinker/heartbeats.go

```diff
@@ -22,7 +22,7 @@ const (
 
 func (svc *sinkerService) sendSingleHeartbeat(t time.Time) {
 	svc.logger.Info("heartbeat")
```

Comment:
this is very noisy in our logs and i don't think provides any value - let's remove while we're working on this